server/tidb_test.go

```diff
@@ -2093,6 +2110,9 @@ func TestTopSQLStatementStats2(t *testing.T) {
 			require.True(t, item.SumDurationNs > 1, sqlStr)
 			foundMap[digest.SQLDigest] = sqlStr
 			tagChecker.checkExist(t, digest.SQLDigest, sqlStr)
+			if strings.Contains(strings.ToLower(sqlStr), "add index") {
+				tagChecker.checkReqExist(t, digest.SQLDigest, sqlStr, tikvrpc.CmdScan)
+			}
```

Comment:
Adding a check for only one kind of SQL smells not good. At least put a comment with the issue number here.